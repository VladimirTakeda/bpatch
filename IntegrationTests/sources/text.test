Some text to replace.
Here should be another result
after processing.