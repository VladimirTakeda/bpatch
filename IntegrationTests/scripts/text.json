{
  "todo": [
    {
      "replace": {
        "line 3": "res3"
      }
    },
    {
      "replace": {
        "line 2": "res2"
      }
    },
    {
      "replace": {
        "line 1": "res1"
      }
    }
  ],
  "dictionary": {
    "text": {
      "line 1": "Some text to replace.",
      "line 2": "Here should be another result",
      "line 3": "after processing.",
      "res1": "Test demonstrates that it is not important if",
      "res2": "todo section in the json file goes first",
      "res3": "and the disctionary is in the end of the script file.\n"
    }
  }
}