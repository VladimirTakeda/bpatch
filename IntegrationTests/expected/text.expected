Test demonstrates that it is not important if
todo section in the json file goes first
and the disctionary is in the end of the script file.
